orca-remote-stage/src/main/kotlin/com/netflix/spinnaker/orca/remote/tasks/StartRemoteStageTask.kt

```diff
@@ -0,0 +1,35 @@
+package com.netflix.spinnaker.orca.remote.tasks
+
+import com.netflix.spinnaker.orca.api.pipeline.RetryableTask
+import com.netflix.spinnaker.orca.api.pipeline.TaskResult
+import com.netflix.spinnaker.orca.api.pipeline.models.ExecutionStatus
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution
+import com.netflix.spinnaker.orca.remote.model.RemoteStageExtensionPayload
+import com.netflix.spinnaker.orca.remote.service.RemoteStageExtensionService
+import java.util.concurrent.TimeUnit
+
+class StartRemoteStageTask(
+  private val remoteStageExtensionService: RemoteStageExtensionService
+) : RetryableTask {
+  override fun getTimeout(): Long = TimeUnit.SECONDS.toMillis(60)
+  override fun getBackoffPeriod(): Long = TimeUnit.SECONDS.toMillis(2)
+
+  override fun execute(stage: StageExecution): TaskResult {
+    val remoteExtension = remoteStageExtensionService.getByStageType(stage.type)
+    val remoteStageExtensionPayload = RemoteStageExtensionPayload(
+      type = stage.type,
+      id = stage.id,
+      pipelineExecutionId = stage.execution.id,
+      context = stage.context
+    )
+
+    remoteExtension.invoke(remoteStageExtensionPayload)
```

Comment:
it might make sense to allow the invoke to return a result that gets added to the context and that would get passed back in subsequent monitor calls

then again that might encourage doing non async things in the invoke, so maybe that is bad

but just thinking of something similar to how we issue a request to clouddriver to start an atomic operation, and that returns a taskId and we can use that taskId to monitor the state of the operation

without supporting something like that, a remote stage is going to have to keep some state on its side that maps pipelineExecutionId to whatever work tracking it is doing